### ISSUE TITLE
Add Spy PDA (Disguise Kit) as option

### ISSUE THREAD

**Issue by RicochetYT** [a-horsey/horsies-viewmodel-editor#4]
The Spy disguise kit (or PDA) is not able to be disabled through this mod, however, it has been done in yttrium's competitive viewmodels (if I'm not mistaken), so it is possible.
![image](https://user-images.githubusercontent.com/36606403/202971252-3eb0bb6a-37ed-4e96-9a7b-b52e0c07b51f.png)
It would be nice to have an option like the Engineer's own PDA.

**Comment by a-horsey** [comment]
Last I checked, it's not included in yttrium's tool. The issue with the pda and all the spy's watches is that they use v_models, compared to all other weapons that use c_models. That means that even if I added a setting for them, you couldn't preload them and have them work in sv_pure 1/2. And even worse, they could cause crashes when joining such servers. So unfortunately these will remain without settings. 

If you really want to and don't mind the issues, it would be pretty easy to make a simple mod to disable it. Just find the disguise kit model, decompile it, open the .qc file, add "origin 9999 9999 9999" and "snap" to all the sequences, compile again and add the new model to custom.
